pages.fr/common/jest.md

```diff
@@ -7,9 +7,13 @@
 
 `jest`
 
-- Exécuter les suites de test des fichiers dont les chemins correspondent aux expressions régulières indiquées :
+- Exécuter les suites de test de fichiers donnés :
```

Comment:
@nicokosi is this change from `des` → `de` intentional?  Just making sure.

Reply:
It's perfect. :+1: